jwst/ami/leastsqnrm.py

```diff
@@ -323,14 +327,14 @@ def populate_antisymmphasearray(deltaps, n=7):
     """
     Populate the antisymmetric fringe phase array:.
 
-    This array takes the form:
+    This array takes the form::
 
-    fringephasearray[0,q+1:] = coeffs[0:6]
-    fringephasearray[1,q+2:] = coeffs[6:11]
-    fringephasearray[2,q+3:] = coeffs[11:15]
-    fringephasearray[3,q+4:] = coeffs[15:18]
-    fringephasearray[4,q+5:] = coeffs[18:20]
-    fringephasearray[5,q+6:] = coeffs[20:]
+        fringephasearray[0, q + 1 :] = coeffs[0:6]
```

Comment:
I don't remember adding spaces around `q + N`. The pre-commit must have done it...